projects/schematics/src/migrations/3_0/component-deprecations/data/close-account-modal.component.migration.ts

```diff
@@ -0,0 +1,14 @@
+import { CLOSE_ACCOUNT_MODAL_COMPONENT } from '../../../../shared/constants';
+import { ComponentData } from '../../../../shared/utils/file-utils';
+
+export const CLOSE_ACCOUNT_MODAL_COMPONENT_MIGRATION: ComponentData = {
+  // projects/storefrontlib/src/cms-components/myaccount/close-account/components/close-account-modal/close-account-modal.component.ts
+  selector: 'cx-close-account-modal',
+  componentClassName: CLOSE_ACCOUNT_MODAL_COMPONENT,
+  removedProperties: [
+    {
+      name: 'userToken',
```

Comment:
change userToken to userToken$

for both the name: and in comment:

also, let's add something non-relevant to this PR, which we missed for `checkout deprecations`
- `ORDER_DETAILS_SHIPPING_COMPONENT` --> change $order to order$ please ;)